frameworks/mxnet/code/inference.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function
```

Comment:
Please add the apache 2.0 license header here

Reply:
License added in all .py files in .`/code`